src/jupytext/__init__.py

```diff
@@ -0,0 +1,29 @@
+"""Read and write Jupyter notebooks as text files"""
+
+from .formats import NOTEBOOK_EXTENSIONS, get_format_implementation, guess_format
+from .jupytext import read, reads, write, writes
+from .reraise import reraise
+from .version import __version__
+
+try:
+    from .contentsmanager import build_jupytext_contents_manager_class
+except ImportError as err:
+    build_jupytext_contents_manager = reraise(err)
+
+try:
+    from .contentsmanager import TextFileContentsManager
+except ImportError as err:
+    TextFileContentsManager = reraise(err)
```

Comment:
Oh the intent here was to display the import error when the `TextFileContentsManager` would be created. I know that sound strange (I was just learning Python at that time). This can probably be removed now.